include/utl/parser/csv.h

```diff
@@ -53,6 +53,16 @@ inline void parse_column(cstr& s, T& arg) {
                                             adjust_for_quote + adjust_for_cr));
 }
 
+inline void unescape_quoted_string(std::string &arg) {
+  auto found_at = 0;
+  while ((found_at = arg.find('"', found_at)) != -1) {
+    if (arg[found_at] == '"' && found_at < static_cast<int>(arg.size()) - 1) {
+      arg.erase(found_at, 1);
+      found_at++; // Skip following character
```

Comment:
```suggestion
      ++found_at; // Skip following character
```

Reply:
If both options do the same, `++i` is always preferable as it doesn't create a temporary. In `Release` mode it probably doesn't matter.